models/exercise.js

```diff
@@ -12,7 +14,7 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
-      Excercise.belongsToMany(Routine, { through: 'RoutineExercises' });
+      Exercise.belongsToMany(Routine, { through: 'RoutineExercises' });
```

Comment:
Can we rename this to only routine and not routineExercise

Reply:
There is already Routine model and table, **RoutineExercises** is the pivot table because there is many to many relationship between **Routine** and **Exercise**